### PR TITLE
Convert tests to use supertest

### PR DESCRIPTION
**Pull request by shadowgate15**
All tests should now use supertest instead of Frisbee.

### CODE REVIEW

File: template/test/web/support.js [hunk 1]
```diff
@@ -1,49 +1,52 @@
...
     'Your support request has been sent successfully.  You should hear from us soon.  Thank you!'
   );
 });
 
 test('fails creating inquiry if last inquiry was within last 24 hours (HTML)', async t => {
-  await global.web.post('/en/support', {
-    body: { email: 'test2@example.com', message: 'Test message!' }
-  });
+  const { web } = t.context;
+  await global.web.post('/en/support')
```
Comment (niftylettuce) [inline template/test/web/support.js:23]: ☝️

**Comment by shadowgate15** [comment]
Wait a little bit on this some of the tests are not written correctly.

**Comment by shadowgate15** [comment]
Currently the snapshot tests fail every time because of the following error in the difference between snapshot and response.

```javascript
        <script>␊
          window.USER = {␊
  -         id: "ifv1MFasogj7xXQlho60bD6F420IXPK3"␊
  +         id: "atZFh3BUUcJo4Fs8uWXmvHDKX_K7sk9l"␊
          };␊
        </script>␊
```

File: template/test/_utils.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
+// Necessary utils for testing
+// Librarires required for testing
+const MongodbMemoryServer = require('mongodb-memory-server').default;
```
Comment (shaunwarman) [inline template/test/_utils.js:3]: 👍

File: template/test/web/index.js [hunk 3]
```diff
@@ -1,35 +1,84 @@
 const test = require('ava');
 
+const { before, beforeEach, afterEach, after } = require('../_utils');
+
+test.before(before);
+test.after.always(after);
+test.beforeEach(beforeEach);
+test.afterEach.always(afterEach);
```
Comment (shaunwarman) [inline template/test/web/index.js:8]: 👍

**Pull request merged**